src/Uno.UI/Controls/CommandBarRenderer.Android.cs

```diff
@@ -63,7 +63,7 @@ private static string ActionBarUpDescription
 
 		public CommandBarRenderer(CommandBar element) : base(element) { }
 
-		protected override Toolbar CreateNativeInstance() => new Toolbar(UI.ContextHelper.Current);
+		protected override Toolbar CreateNativeInstance() => new Toolbar(Uno.UI.ContextHelper.Current);
```

Comment:
Does this come from another PR ?

Reply:
This actually caused errors on my end. Including the namespace fixed the issue.